### PR TITLE
[MOV] *: move all PWA to community

### PR DESCRIPTION
**Pull request by rfr-odoo**
In this commit, we moved all features related to the PWA and the PWA itself to the community.

This includes:
* PWA
* Web Push Notification
* VCARD

Task ID: 3478014

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Great that this finally lands on community. One question: which icon is used for the PWA app?

File: addons/web/controllers/webmanifest.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+from odoo import http
+from odoo.exceptions import AccessError
+from odoo.http import request
+from odoo.tools import ustr, file_open
+
+
+class WebManifest(http.Controller):
+
+    def _get_shortcuts(self):
+        module_names = ['mail', 'crm', 'project', 'note']
```
Comment (rvalyi) [inline addons/web/controllers/webmanifest.py:16]: the 'note' module is probably wrong here as the module has been removed recently https://github.com/odoo/odoo/pull/133101

**Comment by bouvyd** [comment]
> Great that this finally lands on community. One question: which icon is used for the PWA app?

Hello there

At the moment it uses the Odoo icon in the source, not customizable. We don't plan on making that an option right now. We made it easy to rename it should you want to distinguish multiple PWAs.

**Review by xmo-odoo — COMMENTED** [review]
@robodoo override=ci/security

File: addons/web/controllers/webmanifest.py (hunk 1 above)
Comment (AntoniRomera) [inline addons/web/controllers/webmanifest.py:16]: Any plans to add custom modules to the list?

**Review by pparidans — APPROVED** [review]
@robodoo r+

File: addons/web/controllers/webmanifest.py (hunk 1 above)
Comment (bouvyd) [reply addons/web/controllers/webmanifest.py]: not at the moment, though i have it waiting its time in a deep corner of my mind. don't hold your breath though :)

**Review by adr-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**